packages/annotorious-react-manifold/package.json

```diff
@@ -39,9 +39,11 @@
     "vite-tsconfig-paths": "^4.3.1"
   },
   "peerDependencies": {
-    "@annotorious/react": "*",
     "openseadragon": "^3.0.0 || ^4.0.0",
     "react": "16.8.0 || >=17.x || >=18.x",
     "react-dom": "16.8.0 || >=17.x || >=18.x"
+  },
+  "dependencies": {
+    "@annotorious/react": "*"
```

Comment:
Was moved to a `dependencies` as an essential package w/o which that package won't properly run or build. The "*" will also help to avoid versions duplication in the consuming application